### PR TITLE
update idp section with new admin ui

### LINKED ISSUE #1489: Integrating identity providers chapter
### Description

Update this Server Administration Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1489

### CODE REVIEW

File: server_admin/topics/identity-broker/default-provider.adoc [hunk 1]
```diff
@@ -8,7 +8,13 @@
 .Procedure
 . Click *Authentication* in the menu.
 . Click the *Browser* flow.
-. Select *Identity Provider Redirector* from the drop-down list. 
+ifeval::[{project_community}==true]
+. Click *⚙️* (gear icon) of the *Identity Provider Redirector* row.
```
Comment (andymunro) [inline server_admin/topics/identity-broker/default-provider.adoc]:
```suggestion
. Click the gear icon  *⚙️* on the *Identity Provider Redirector* row.
```

File: server_admin/topics/identity-broker/default-provider.adoc [hunk 2]
```diff
@@ -8,7 +8,13 @@
...
+endif::[]
+ifeval::[{project_product}==true]
+. Select *Identity Provider Redirector* from the drop-down list.
+. Set *Default Identity Provider* to the identity provider you want to redirect users to.
```
Comment (andymunro) [inline server_admin/topics/identity-broker/default-provider.adoc]: This step is already included below.

File: server_admin/topics/identity-broker/configuration.adoc [hunk 3]
```diff
@@ -11,7 +11,12 @@ The foundations of the identity broker configuration are identity providers (IDP
 .Identity Providers
 image:{project_images}/identity-providers.png[Identity Providers]
 +
+ifeval::[{project_community}==true]
+. Select card of the identity provider you want to add. {project_name} displays the configuration page for the identity provider you selected.
```
Comment (andymunro) [inline server_admin/topics/identity-broker/configuration.adoc]:
```suggestion
. Select an identity provider. {project_name} displays the configuration page for the identity provider you selected.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, looks good. Just a few changes @edewit.

**Comment by andymunro** [comment]
Thanks, @edewit.  Another one ready to merge, @ssilvert 

Thanks also for replacing te GitHub screen shot with one for GitLab.

**Review by andymunro — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**